src/main/protobuf/v1/token.proto

```diff
@@ -11,6 +11,8 @@ message TokenV1 {
   repeated PartyRelationshipBindingV1 legals = 2;
   required string validity = 3;
   required string checksum = 4;
+  required string contractPath = 5;
+  required string contractVersion = 6;
```

Comment:
may it have sense to group these fields under the one object?

Reply:
please, consider that these become token claims. Could be annoying to have nested ones?

Reply:
not sure, annoying referred to which issue in particular?

Reply:
I don't know, I use to see flat jwt claims 🤷🏻‍♂️